testnets/qwoyntestnet/chain.json

```diff
@@ -0,0 +1,90 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "qwoyntestnet",
+  "status": "live",
+  "network_type": "testnet",
+  "website": "https://qwoyn.studio/",
+  "pretty_name": "Qwoyn",
+  "chain_id": "higgs-boson-1",
+  "bech32_prefix": "qwoyn",
```

Comment:
Will be able to proceed once registered to that repo

Reply:
SLIP_0173 will have all the chains listed, with their bech32 prefix specified. No number is used.

Reply:
Registered now